docs/manual/contributing/getting-started.md

```diff
@@ -0,0 +1,43 @@
+# Getting started {#sec-contrib-getting-started}
+
+If you have not previously forked Home Manager then you need to do that
+first. Have a look at GitHub's [Fork a
+repo](https://help.github.com/articles/fork-a-repo/) for instructions on
+how to do this.
+
+Once you have a fork of Home Manager you should create a branch starting
+at the most recent `master` branch. Give your branch a reasonably
+descriptive name. Commit your changes to this branch and when you are
+happy with the result and it fulfills [Guidelines](#sec-guidelines) then
+push the branch to GitHub and [create a pull
+request](https://help.github.com/articles/creating-a-pull-request/).
+
+Assuming your clone is at `$HOME/devel/home-manager` then you can make
+the `home-manager` command use it by either
+
+1.  overriding the default path by using the `-I` command line option:
+
+    ``` console
+    $ home-manager -I home-manager=$HOME/devel/home-manager
```

Comment:
The generated code is slightly different so the custom CSS that makes the `$` non-selectable doesn't trigger. The code https://git.sr.ht/~rycee/nmd/tree/master/item/static/style.scss#L297-300 should probably be changed to something like

``` css
.programlisting.language-shell .hljs-meta.prompt_ {
    user-select: none;
}
```

I can have a look at fixing that.

Reply:
If you update the `nmdSrc` rev to `f18defadcc25e69e95b04493ee02682005472255`, then selection issue should now be resolved.

Reply:
Yeah, I think adding a commit with

``` patch
diff --git a/docs/default.nix b/docs/default.nix
index 0c2c5516..ecef00c3 100644
--- a/docs/default.nix
+++ b/docs/default.nix
@@ -9,8 +9,8 @@ let
 
   nmdSrc = fetchTarball {
     url =
-      "https://git.sr.ht/~rycee/nmd/archive/824a380546b5d0d0eb701ff8cd5dbafb360750ff.tar.gz";
-    sha256 = "0vvj40k6bw8ssra8wil9rqbsznmfy1kwy7cihvm13rajwdg9ycgg";
+      "https://git.sr.ht/~rycee/nmd/archive/f18defadcc25e69e95b04493ee02682005472255.tar.gz";
+    sha256 = "14fazjqbc6sfsxycwadfhcymi37xg5wmrlvg3jb8d4mpbcdwfwz4";
   };
 
   nmd = import nmdSrc {
```

is worth doing regardless :-)